tasks/self-introduction/self-introduction.md

```diff
@@ -1,59 +1,59 @@
+1. Record your presentation in English. It is desirable that your face is in the frame (for recording, you can use a webcam, phone camera, video camera, laptop camera, etc.).
+2. Upload the video to your personal [YouTube](https://www.youtube.com/) account. You can make a video available only via a link - to do this, in the video settings, set [Параметры доступа > Доступ по ссылке (Visibility > Unlisted)](https://support.google.com/youtube/answer/157177?co=GENIE.Platform%3DDesktop&hl=en).
+3. In your **private** repository (using a link like `https://github.com/rolling-scopes-school/your-github-account-JSFE2022Q3`) create a `self-introduction` branch from the `main` branch.
+4. In the created branch `self-introduction` create a folder `self-introduction` with a file `index.md` inside.
+5. Add a link to your YouTube video to the `index.md` file
+6. Add a text transcript of your presentation below the link. The text should repeat what was said in your presentation, but may not be verbatim - you can edit the text by removing unnecessary interjections, parasitic words, etc. from it.
+7. Use gh-pages for deployment [Как сделать деплой задания из приватного репозитория школы](https://docs.rs.school/#/private-repository?id=Как-сделать-деплой-задания-из-приватного-репозитория-школы)
+8. Make a Pull Request from the `self-introduction` branch to the `main` branch. The Pull Request must be named the same as the task, **Self-Introduction**, and conform to the [Requirements for Pull Requests](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr). As a screenshot of the result, you can attach a screenshot of the video, and as a link to the deployment use a link like `https://rolling-scopes-school.github.io/your-github-account-JSFE2022Q3/self-introduction/`. **Don't merge the Pull Request**.
 
-> ### Важно
+> ### Improtant
 >
-> - Проверьте, что видео и текст доступны по ссылкам и для просмотра не требуется никаких дополнительных разрешений. Проще всего проверить это, открыв обе ссылки в режиме инкогнито в браузере.
-> - Мы не просим вас загружать видео в открытый доступ. Вы можете [включить доступ к ролику только по ссылке](#работа-с-youtube). Такое видео смогут просмотреть только пользователи, у которых есть нужный URL.
+> - Check that the video and text are available via links and that no additional permissions are required to view. The easiest way to check this is by opening both links in incognito mode in a browser.
+> - You can [enable access to the video only by link](#work-from-youtube). Only users who have the desired URL will be able to view such a video.
```

Comment:
There is no such id #work-from-youtube. I guess, it should be an anchor to the step 2 of the Task implementation or to the How to work with YouTube